src/elements/content-sidebar/activity-feed/activity-feed/ActivityFeed.js

```diff
@@ -202,6 +205,19 @@ class ActivityFeed extends React.Component<Props, State> {
         const isEmpty = this.isEmpty(this.props);
         const isLoading = !this.hasLoaded();
 
+        const activeEntry =
+            Array.isArray(feedItems) &&
+            feedItems.find(({ id, type }) => id === activeFeedEntryId && type === activeFeedEntryType);
+
+        const errorMessageByEntryType = {
+            comment: messages.commentMissingError,
+            task: messages.taskMissingError,
+        };
+
+        const errorMessage = activeFeedEntryType ? errorMessageByEntryType[activeFeedEntryType] : undefined;
+
+        const isErrorVisible = !isLoading && activeFeedEntryType && !activeEntry;
```

Comment:
activeEntry is used for an entry in the feed item that matches (this would not render an error). In this case, I prefixed errorMessage and isErrorVisible with inlineFeedItem since I think that is more relevant.